lang/solidity/Portfile

```diff
@@ -24,36 +23,20 @@ long_description    Solidity is a statically typed, contract-oriented, \
                     high-level language for implementing smart contracts \
                     on the Ethereum platform.
 homepage            https://soliditylang.org
-master_sites        https://github.com/ethereum/solidity/releases/download/v${github.version}/
 
-checksums           rmd160  df322fa402d7fc3d1c6643c0136a44251189ce28 \
-                    sha256  9bc195e1695f271b65326c73a12223377d52ae4b4fee10589c5e7bde6fa44194 \
-                    size    3239230
+checksums           rmd160  14a80ba6315744ec6892405a91b0ce22195cb78f \
+                    sha256  def2ab7c877fcd16c81a166cdc5b99bfabcee7e8d505afcce816e9b1e451c61a \
+                    size    3386993
 
-set port_libfmt     libfmt9
-
-depends_build-append \
-                    port:${port_libfmt} \
-                    port:range-v3
-depends_lib         port:jsoncpp
-
-cmake.module_path-append \
-                    ${prefix}/lib/${port_libfmt}/cmake
+depends_lib-append  port:boost
 
 compiler.cxx_standard \
                     2017
 
 configure.args      -DTESTS:BOOL=OFF \
                     -DBoost_USE_STATIC_LIBS:BOOL=OFF \
-                    -DWITH_Z3:BOOL=OFF \
-                    -DWITH_CVC4:BOOL=OFF \
-                    -DUSE_LD_GOLD:BOOL=OFF \
-                    -DRANGE_V3_INCLUDE_DIR:PATH="${prefix}/include" \
-                    -DJSONCPP_INCLUDE_DIR:PATH="${prefix}/include/jsoncpp" \
-                    -DJSONCPP_LIB_DIR:PATH="${prefix}/lib"
```

Comment:
ideally we don't want to use vendored packages but the ones that MacPorts installs. Is the port not using any of the dependencies listed here anymore?

Reply:
The latest version of the required dependencies is managed through CMake, which requires a special version and may not be suitable for use with MacPorts.

Reply:
I'm not sure what you mean. It either needs the dependencies or it doesn't. So presumably after this change it is using vendored versions these packages, that is not what we want. Specifying with CMake variables and point to ${prefix} should make sure the MacPorts versions are used. 

Please check this again as well as the boost dependency, that should ideally work with the PortGroup.